.env.example

```
@@ -1,2 +1,5 @@
 VITE_COINGECKO_API_KEY=""
 VITE_ETHERSCAN_API_KEY=""
+
+VITE_VITE_INFURA_API_KEY=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)